pagerduty/resource_pagerduty_event_orchestration_path_global_test.go

```diff
@@ -0,0 +1,739 @@
+package pagerduty
```

Comment:
The test steps are very similar to those in `pagerduty/resource_pagerduty_event_orchestration_path_service_test.go`, we make sure we can add/update/delete individual properties.